packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -82,19 +84,28 @@ class MediaQueryData {
     return size.width > size.height ? Orientation.landscape : Orientation.portrait;
   }
 
+  final TargetPlatform _platform;
+
+  /// The platform widgets should adapt to target.
+  ///
+  /// Defaults to the current platform.
+  TargetPlatform get platform => _platform ?? defaultTargetPlatform;
```

Comment:
Why do you default it here as well as earlier? Surely this can't be null at this point.